src/assets/src/models.ts

```diff
@@ -33,6 +33,7 @@ export interface Meeting {
     backend_type: "bluejeans"|"zoom"|"inperson";
     backend_metadata?: BluejeansMetadata|ZoomMetadata;
     created_at: string;
+    status: 0 | 1 | 2;  // 0 is UNASSIGNED, 1 is ASSIGNED, 2 is STARTED
```

Comment:
I think an enum could be used here.

Reply:
So like this?

```
enum MeetingStatus {
   0,
   1,
   2
}
```

Reply:
I was thinking:
```
enum MeetingStatus {
        UNASSIGNED = 0,
        ASSIGNED = 1,
        STARTED = 2,
}
```
Which results in (from node repl):
```
> MeetingStatus
{
  '0': 'UNASSIGNED',
  '1': 'ASSIGNED',
  '2': 'STARTED',
  UNASSIGNED: 0,
  ASSIGNED: 1,
  STARTED: 2
}
```

Reply:
Mmm, yes, so like this? https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/240/commits/8fec90a9f5db8a79cefca5d30a7a48120aa9e3f2